src/Tec/Plugin.php

```diff
@@ -100,19 +98,108 @@ public function register() {
 		}
 
 		// Do the settings.
-		// TODO: Remove if not using settings
 		$this->get_settings();
 
 		// Start binds.
 
+		// Remove reCAPTCHA v2
+		add_filter('tribe_community_events_captcha_plugin', '__return_null' );
 
+		$this->maybe_do_recaptcha_v3();
 
 		// End binds.
 
 		$this->container->register( Hooks::class );
 		$this->container->register( Assets::class );
 	}
 
+	/**
+	 * Replace reCAPTCHA v2 with v3 if there is a license key set.
+	 *
+	 * @return void
+	 *
+	 * @since 1.0.0
+	 */
+	function maybe_do_recaptcha_v3() {
+		$ce_options = \Tribe__Events__Community__Main::getOptions();
+		$option_key = $this->get_options_prefix() . '_site_key';
+		$recaptcha_key = $ce_options[$option_key];
+
+		if ( $recaptcha_key != '' ) {
```

Comment:
Let's use strict comparison

```suggestion
		if ( $recaptcha_key !== '' ) {
```